app/models/data_resources/open_weather_map.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+class OpenWeatherMap < ApplicationRecord
+
+  store :data,
+        accessors: %i[
+          lat
+          lon
+          current
+          hourly
+          daily
+          alerts
```

Comment:
was sind das genau für eigenschaften, die so eine open weather map data hat?
kannst du zu jeden ein stichpunkt ergänzen in einem kommentar?

Reply:
jein, das kann vermutlich Julian tun, ich habe mir die Daten nicht näher angesehen. Vorgabe war von Julian genau diese Daten zu erhalten.

Reply:
@digorath wollen wir irgendwo dokumentieren, wie wir daten abfragen, was für daten wir erhalten und welche daten wir für die mobile app verwenden?

Reply:
Ich habe es auf meine todo liste für confluence gesetzt.
sobald ich dazu komme werde ich eine seite dafür aufsetzen und ein paar worte zu den genutzten typen im backend und in der app sagen.